<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyProfileDialog</class>
 <widget class="QDialog" name="PyProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>587</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>3</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Profile Results</string>
  </property>
  <property name="whatsThis">
   <string>&lt;b&gt;Profile Results&lt;/b&gt;
&lt;p&gt;This dialog shows the profile results.&lt;/p&gt;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QTreeWidget" name="resultList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Profile Results&lt;/b&gt;
&lt;p&gt;This list shows the profile results. There are several actions available via the context menu.&lt;/p&gt;</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Nr. Calls</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Total Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Tot. Time / Call</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cumulative Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cum. Time / Call</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Filename</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Line</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Function</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="summaryList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Summary&lt;/b&gt;
&lt;p&gt;This shows some overall profile data. There are several actions available via the context menu.&lt;/p&gt;</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="itemsExpandable">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Summary</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>#</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="checkProgress">
     <property name="toolTip">
      <string>Shows the progress of the profile data calculation</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="format">
      <string>%v/%m Files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>resultList</tabstop>
  <tabstop>summaryList</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
